backend/src/chats/chat/entities/chat.entity.ts

```diff
@@ -52,7 +52,7 @@ export class Chat {
 	@Exclude()
 	password: string;
 
-	@OneToMany(() => Message, (message) => message.chat_id)
+	@OneToMany(() => Message, (message) => message.chat)
 	messages: Message[];
```

Comment:
what would be the difference between chat_id or chat? or is it just different naming?

Reply:
I renamed the relationship in messages from `chat_id` to `chat` and added a `chat_id` column that actually stores the relationid (see message.entity)